test/e2e/tests/custom-token-send-transfer.spec.js

```diff
@@ -0,0 +1,445 @@
+          windowHandles,
+        );
+        await driver.waitForSelector({ text: '1.5 TST', tag: 'h1' });
+        await driver.clickElement({ text: 'Confirm', tag: 'button' });
+        await driver.switchToWindow(extension);
+        // checks if transaction is done correctly from extension
+        await driver.clickElement({ tag: 'button', text: 'Activity' });
+        await driver.findElements('.transaction-list__pending-transactions');
+        await driver.waitForSelector(
+          {
+            css: '.transaction-list-item__primary-currency',
+            text: '-1.5 TST',
+          },
+          { timeout: 10000 },
+        );
+        await driver.clickElement('.transaction-list-item__primary-currency');
+
+        const transactionAmounts = await driver.findElements(
+          '.currency-display-component__text',
+        );
+        const transactionAmount = transactionAmounts[0];
+        assert(
+          await transactionAmount.getText(),
+          '1.5 TST',
+          'Token amount is not correct',
+        );
+      },
+    );
+  });
+
+  it('customizes gas, submits the transaction and finds the transaction in the transactions list', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'connected-state',
+        ganacheOptions,
+        smartContract,
+        title: this.test.title,
+        failOnConsoleError: false,
+      },
+      async ({ driver, contractRegistry }) => {
+        const contractAddress = await contractRegistry.getContractAddress(
+          smartContract,
+        );
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+        // create token
+        await driver.openNewPage(
+          `http://127.0.0.1:8080/?contract=${contractAddress}`,
+        );
+        let windowHandles = await driver.getAllWindowHandles();
+        const extension = windowHandles[0];
+        await driver.findClickableElement('#deployButton');
+
+        // add token from dapp
+        await driver.clickElement({
+          text: 'Add Token to Wallet',
+          tag: 'button',
+        });
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.clickElement({ text: 'Add token', tag: 'button' });
+        windowHandles = await getWindowHandles(driver, 2);
+        // transfer token from dapp
+        await driver.clickElement({ text: 'Transfer Tokens', tag: 'button' });
+        await driver.waitUntilXWindowHandles(3);
+        windowHandles = await driver.getAllWindowHandles();
+
+        await driver.switchToWindowWithTitle(
+          'MetaMask Notification',
+          windowHandles,
+        );
+        await driver.waitForSelector({ text: '1.5 TST', tag: 'h1' });
```

Comment:
I think we can remove this line

Reply:
If I remove it I'm getting error on the line after it : 
 _ElementClickInterceptedError: Element <button class="button btn--rounded btn-primary page-container__footer-button"> is not clickable at point (266,567) because another element <div class="loading-overlay"> obscures it_

So I put this waitForSelector here since it seems like it needs a little bit more time to load popup properly. I could use delay but we all agreed to avoid it. Hope this makes sense.

Reply:
Thank you, that make sense! Let's leave it as it is then! @mirjanaKukic